src/projects/projects-page.html

```diff
@@ -33,10 +33,12 @@
         flex: 1;
       }
       project-overview-page {
-        max-width: 400px;
         overflow-y: scroll;
         overflow-x: hidden;
       }
+      project-pull-requests {
+        max-width: 630px;
```

Comment:
This does not look on my screen. The previous on `project-overview-page` was better imo.